styles.css

```diff
@@ -78,15 +82,103 @@ h2 {
   color: #f0e3ca;
 }
 
-.h3-container {
+h3 {
+  font-weight: 700;
+  font-size: 2rem;
+  text-align: center;
+  color: #ff8303;
+}
+
+.p-container {
   width: 18em;
   margin-left: auto;
   margin-right: auto;
 }
 
-h3 {
+p {
   font-weight: 400;
   font-size: 1.5rem;
   text-align: justify;
   color: #f0e3ca;
+  background-color: transparent;
+}
+
+hr {
+  color: #111;
+}
+
+/* This code is for the logo at the beging of the About Me section */
+#aboutMeLogo {
+  margin: 0 auto;
+  height: 4em;
+}
+
+.skillsContainer {
+  display: grid;
+  justify-content: center;
+}
+
+.skillsContainer > div {
+  display: inline-flex;
+  margin: 1em;
+  padding-left: 5em;
```

Comment:
😃 📱  The skills section looks good, and the behavior of icons and text is nice. I recommend setting this padding for even better results.

Reply:
Changes applied using vw unit instead.